Adds the Minutemen Corvus-class.

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
this pull request adds the "Corvus-class", a minutemen response vessel. 
 its main purpose is to detain criminals, patrol the sector, and respond to distress calls. 
 its decently armed, containing a vector, two m1911's, a flamethrower, a colt revolver and some general security gear. it however in return, lacks in most other departments.

![image](https://user-images.githubusercontent.com/79304582/188908645-c0483b27-de49-4492-90dd-45e3596f0779.png)


![2022 09 07-16 47 53](https://user-images.githubusercontent.com/79304582/188908769-749efc84-37d9-4ab8-9775-10af46633b04.png)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
currently, the only minutemen ship we have is the Carina-class, which is pretty sad. especially for a faction with as much content as the minutemen. 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Added the Corvus-class
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
adds the dmm and Json

boop di boop, i make'a the pr

spockye committed:
misc fixes

fixes the airtank dir
fixes the armoury
clothing misc
other misc stuff

spockye committed:
fixes linters

added capital to last part
changes the back windows to normal reinforced

spockye committed:
piping errors

hopefully fixes all of em

ZephyrTFA commented:
![image](https://user-images.githubusercontent.com/12817816/187518938-d8ae7304-dfbf-4388-baf8-4df9eff4ea5d.png)

spockye commented:
> ![image](https://user-images.githubusercontent.com/12817816/187518938-d8ae7304-dfbf-4388-baf8-4df9eff4ea5d.png)

![image](https://user-images.githubusercontent.com/79304582/187519029-e7c97cd7-2004-4740-a9e9-79b4bc6d3d66.png)

spockye committed:
watch as i murder a innocent airlock

spockye committed:
adds outpost console

spockye committed:
spare plasma/notice board

Latentish commented:
YOOOOO

spockye committed:
added a flamethrower, removed inducer

Latentish commented on _maps/shuttles/shiptest/minutemen_corvus.dmm:
```diff
@@ -0,0 +1,3147 @@
+/obj/item/clothing/head/cowboy/sec/minutemen,
+/obj/item/clothing/suit/toggle/lawyer/minutemen,
+/obj/item/clothing/shoes/combat,
+/obj/item/radio/headset/heads/captain/alt,
```
Use faction headsets.

Latentish commented on _maps/shuttles/shiptest/minutemen_corvus.dmm:
```diff
@@ -0,0 +1,3147 @@
+	},
+/obj/item/storage/belt/sabre,
+/obj/item/clothing/head/beret/hop{
+	name = "captain beret"
```
The Minutemen have their own uniforms, don't use this beret please.

Latentish commented on _maps/shuttles/shiptest/minutemen_corvus.dmm:
```diff
@@ -0,0 +1,3147 @@
+	name = "\proper captain's locker";
+	req_access_txt = "20"
+	},
+/obj/item/storage/belt/sabre,
```
This sabre is an NT thing. The Luxemburg has it because they're a trade vessel so it's assumed that it's not from the syndicate.

Latentish reviewed (changes requested):
- The industrial hardsuit containment units would look really nice here.
![image](https://user-images.githubusercontent.com/55075747/188250401-38b47ed3-e330-4b58-a764-06b678f1e5b7.png)

- You could go the extra mile and make the toilet in the bathroom a secret subtype and put pills or wine in it.

- Medibots are kinda busted. If you replace it with a robot arm so they can make the rest of hte parts themselves, that would be good.

- You should replace the windoors in front of the plasma engine prechargers with normal windows.

- I'd like a full list of the weapons on this ship.

Latentish commented on _maps/configs/minutemen_corvus:
```diff
@@ -0,0 +1,28 @@
+{
```
It's good practice to add the schema here.

spockye replied:
where would i find the "schema"?

spockye committed:
removes and replaces

industrial suit container
removed beret
removed saber
replaced headset
removed medibot

spockye committed:
windoor to reinforced plasma glass

spockye committed:
adds schema

who what why whahaaahhahahhahwhehhwhwh

spockye committed:
adds some stuff

adds some spare plasma (10=

adds a rd console and server

spockye committed:
wait, i have dementia

i already had some from before.... i'll remove the old one in spite of my younger self

spockye committed:
updates some of the doors

Missing names!

spockye committed:
misc additions

adds a 38. for the captain
adds blueprints
adds a gps
adds 1000 credits on the table
removes duplicate eyepatch

Latentish commented:
Update your images and get me the list of weapons and I'll give you a review.

spockye commented:
woop woop

spockye committed:
misc fixes and changes

updates button names
removes stacked hardsuit containers
adds a super secret bee plush

spockye committed:
Merge branch 'master' into Minutemen_Lepus

Latentish commented:
These are why this is failing. These items need to be updated.
![image](https://user-images.githubusercontent.com/55075747/198582857-4c64244e-cdcd-46f5-8afc-09b3548ab86e.png)

spockye committed:
misc fixes

adds holofields
switches riot suit for minutemen riot suit
fixes clothing

Latentish commented:
test marg

spockye commented:
(should be good to merge, unless there are any other issues i haven't been informed of)
